core/src/main/java/org/infinispan/util/logging/Log.java

```diff
@@ -1328,4 +1328,8 @@ void asyncStoreShutdownTimeoutTooHigh(long configuredAsyncStopTimeout,
    @LogMessage(level = WARN)
    @Message(value = "No indexing service provider found for indexed filter of type %s", id = 363)
    void noFilterIndexingServiceProviderFound(String filterClassName);
+
+   @Message(value = "Attempted to register cluster listener of class %s, but listener is annotated as only observing pre events! " +
+           "Ignoring listener.", id = 364)
```

Comment:
Since and exception in being thrown nothing gets ignored. Please fix the message.